packages/preview/kdl-unofficial-template/0.1.0/typst.toml

```diff
@@ -0,0 +1,12 @@
+[package]
+name = "kdl-unofficial-template"
+version = "0.1.0"
+authors = ["Martin Heuschober"]
+license = "MIT"
+entrypoint = "lib/lib.typ"
+description = "An unofficial template for community scenarios for the TTRPG 'KULT: divinity lost'"
+
```

Comment:
As reported by the CI, you should include at least one category for your package.